CHANGELOG.rst

```diff
@@ -15,6 +15,11 @@ Change Log
 
 Unreleased
 ----------
+
+[4.16.0]
+---------
+* fix: fix the PATCH method of ``EnterpriseCourseEnrollmentView``.
```

Comment:
super nit: if the version bump is to 4.16, should the commit prefix be `feat`? Or release it as a patch fix (4.15.13).